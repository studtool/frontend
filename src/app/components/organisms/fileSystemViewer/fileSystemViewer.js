import React, {Component} from 'react';
import {Button} from '../../atoms/button/button.js';
import {InputText} from '../../molecules/inputText/inputText.js';

export class FileSystemViewer extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        const {
            qa = false,
            data = {},
            handleSubmit = null,
        } = this.props;

        return (
            <>
                lol
            </>
        );
    };
}
