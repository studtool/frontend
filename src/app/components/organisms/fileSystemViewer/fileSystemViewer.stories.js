import * as React from 'react';
import {storiesOf} from '@storybook/react';
import {FileSystemViewer} from 'App/components/organisms/fileSystemViewer/fileSystemViewer';

storiesOf('File system viewer', module)
    .add('default', () => (
        <FileSystemViewer/>
    ))
    .add('with qa', () => (
        <FileSystemViewer qa={true}/>
    ))
