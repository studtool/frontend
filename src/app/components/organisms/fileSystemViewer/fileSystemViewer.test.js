import React from 'react';
import {FileSystemViewer} from './fileSystemViewer.js';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

test('signUp Form test', () => {
    const stubSubmit = () => {};
    const stubChange = () => {};
    const data = {
        email__errorMessage: '',
        password__errorMessage: '',
        passwordRepeat__errorMessage: '',
    };

    const tree = renderer.create(
        <FileSystemViewer
            handleSubmit={stubSubmit}
            handleChange={stubChange}
            data={data}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
