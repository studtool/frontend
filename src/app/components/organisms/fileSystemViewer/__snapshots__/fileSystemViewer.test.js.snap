// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signUp Form test 1`] = `
Array [
  <div>
    <button
      onClick={[Function]}
    >
      +
    </button>
    <div>
      <div
        contentEditable={true}
        onKeyDown={[Function]}
      >
        hi
      </div>
    </div>
  </div>,
  <div>
    <button
      onClick={[Function]}
    >
      +
    </button>
    <div>
      <div
        contentEditable={true}
        onKeyDown={[Function]}
      >
        lol
      </div>
    </div>
  </div>,
  <div>
    <button
      onClick={[Function]}
    >
      +
    </button>
    <div>
      <div
        contentEditable={true}
        onKeyDown={[Function]}
      >
        kek
      </div>
    </div>
  </div>,
]
`;
